--- a/cmake/tpl/SundialsPOSIXTimers.cmake
+++ b/cmake/tpl/SundialsPOSIXTimers.cmake
@@ -43,7 +43,7 @@
 
     # Create a CMakeLists.txt file which will generate the test executable
     file(WRITE ${POSIXTest_DIR}/CMakeLists.txt
-      "CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)\n"
+      "CMAKE_MINIMUM_REQUIRED(VERSION 3.5)\n"
       "PROJECT(posixtimerstest C)\n"
       "SET(CMAKE_VERBOSE_MAKEFILE ON)\n"
       "SET(CMAKE_C_COMPILER \"${CMAKE_C_COMPILER}\")\n"
